shoop/core/models/_contacts.py

```diff
@@ -170,7 +207,8 @@ def delete(self, *args, **kwargs):
 
     @property
     def groups(self):
```

Comment:
@tulimaki You're right. We totally missed this with @Pikkupomo yesterday. Please add a docstring to the AnonymousContact.groups property which clarifies this, e.g.

```python
@property
def groups(self):
    """
    Contact groups accessor for anonymous contact.

    The base class already has a `groups` property via `ContactGroup`
    related_name, but this overrides it for `AnonymousContact` so that
    it will return a queryset containing just the anonymous contact
    group rather than returning the original related manager, which
    cannot work since `AnonymousContact` is not in the database.

    This allows to use statements like this for all kinds of contacts,
    even `AnonymousContact`::

        some_contact.groups.all()

    :rtype: django.db.QuerySet
    """
```